### DISCUSSION TITLE
scatter_softmax

### DISCUSSION THREAD

**Discussion by xnuohz** [pyg-team/pytorch_geometric#2345] [Q&A]
Does `scatter_softmax` compute weights normalized by src nodes or dst nodes? If so, any docs for this function? Thanks:)

**Comment by rusty1s** [comment]
It depends on which `index` you are passing to the `softmax` function. In general, we normalize across incoming messages, i.e., by destination nodes.

**Reply by xnuohz** [reply]
Ok, thanks for your reply.
